inspirehep/modules/search/templates/search/search.html

```diff
@@ -83,9 +83,16 @@
           </div>
           {%- endif %}
           {%- endblock search_sort %}
-          </div>
         </div>
       </div>
+      {% if request.args.get('cc') in ["jobs"] %}
```

Comment:
We now have a [search template per collection](https://github.com/inspirehep/inspire-next/tree/invenio3/inspirehep/modules/search/templates/search) so you can override the block and personalize it in the jobs template.